rpgpy/nc.py

```diff
@@ -152,3 +188,21 @@ def _get_measurement_date(file: netCDF4.Dataset) -> list:
     date = utils.rpg_seconds2date(ma.min(time), date_only=True)
     assert_array_equal(date, utils.rpg_seconds2date(ma.max(time), date_only=True))
     return date
+
+def _generator_files(dir: str = cwd):
+    """"Function that creates a generator with filepaths
+    of leve0 and level1 files.
+
+    """
+
+    includes = ('.LV0', '.LV1', '.lv0', 'lv1')
+    
+    for subdir, dirs, files in sorted(os.walk(dir)):
+        for file in files:
+            if file.endswith(includes):
+                yield os.path.join(subdir, file)
+
+def _new_filename(filepath):
+    filename = filepath.split('/')[-1] + '.nc'
```

Comment:
This internal function takes the filepath - which is the [next](https://docs.python.org/3/library/functions.html#next) element of the generator and returns the old filename and appends `.nc` to it!